railties/lib/rails/generators/rails/app/templates/app/views/layouts/application.html.erb.tt

```diff
@@ -5,6 +5,11 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <meta name="apple-mobile-web-app-capable" content="yes">
     <meta name="mobile-web-app-capable" content="yes">
+    <%- unless options[:skip_hotwire] || options[:skip_javascript] -%>
+    <meta name="view-transition" content="same-origin">
+    <meta name="turbo-refresh-method" content="morph">
+    <meta name="turbo-refresh-scroll" content="preserve">
```

Comment:
Should we use the `turbo_refreshes_with` helper here? It is [defined in turbo-rails](https://github.com/hotwired/turbo-rails/blob/0e42702df40698521cc7e506fea927bc3a2a08ab/app/helpers/turbo/drive_helper.rb#L70C1-L73C6).